python/cudf/cudf/core/column/categorical.py

```diff
@@ -782,8 +782,6 @@ def to_pandas(
         arrow_type: bool = False,
     ) -> pd.Index:
         if nullable:
-            return super().to_pandas(nullable=nullable, arrow_type=arrow_type)
-        elif arrow_type:
             raise NotImplementedError(f"{arrow_type=} is not implemented.")
```

Comment:
```suggestion
            raise NotImplementedError(f"{nullable=} is not implemented.")
```
Base on what was removed above

Reply:
@galipremsagar could you take a look at this?

Reply:
Yup, updated this. We are not changing the behavior for `categorical` columns or types in this PR. There is additional complexity with supporting `pd.ArrowDtype(pa.dict())`, so I'm punting that to another PR for `25.08`. Coming to this method, on `branch-25.06` we throw an exception when either one of `nullable` or `arrow_type` are `True`, I'm preserving that behavior in this PR.